cmd/deploy/deploy.go

```diff
@@ -278,7 +279,7 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		return err
 	}
 	oktetoLog.Debug("found okteto manifest")
-	if deployOptions.Manifest.Deploy == nil {
+	if deployOptions.Manifest.Deploy == nil || (deployOptions.Manifest.Deploy.Commands == nil && deployOptions.Manifest.Deploy.ComposeSection == nil && deployOptions.Manifest.Deploy.Divert == nil) {
```

Comment:
Could we create a function on `DeployInfo` that checks if is has any deploy command? This if clausure is difficult to read 🙂

Reply:
i've reverted this change, we only checkc if deploy is nil, otherwise, we deploy whatever is at the deploy manifest.

When deploying only endpoints, the endpoints get deployed, but not shown at the UI. When i go `okteto endpoints list` i get the endpoint there

Reply:
reverted this change